index.html

```diff
@@ -1,76 +1,273 @@
+      </div>
+    <!--CARD 3-->
+
+
+    <div class="card">
+      <div class="margin">
+      <div class="card-img">
+        <img class="card-img1" src="./image-folder/Snapshoot Portfolio (3).png" alt="">
+
+      </div>
+      <div class="project-heading">
+        <h2>Tonic</h2>
+        <ul class="canopy">
+          <li class="senior">CANOPY</li>
+          <li><img src="./image-folder/Counter.png" alt=""></li>
+          <li>Back End Dev</li>
+          <li><img src="./image-folder/Counter.png" alt=""></li>
+          <li>2015</li>
+        </ul>
+        <p class="canopy-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="canopy-css">
+          <ul class="canopy-list">
+            <li class="list">HTML</li>
+            <li class="list">CSS</li>
+            <li class="list0">JavaScript</li>
+          </ul>
 
-      <div class="socials">
-        <div class="connect">
-          <a href="#" class="link">LETS CONNECT</a>
         </div>
+        <button class="btn">See project</button>
+      </div>
+    </div>
+    </div>
+
+    <!--CARD 4-->
 
-        <div class="socials-icon">
-          <ul class="social-pic">
-            <li>
-              <a href=""><img src="./image-folder/Vector (3).svg" alt="" /></a>
-            </li>
-            <li>
-              <a href=""><img src="image-folder/Linkedin icon.svg" alt="" /></a>
-            </li>
-            <li>
-              <a href=""><img src="image-folder/Vector (2).svg" alt="" /></a>
-            </li>
-            <li>
-              <a href=""><img src="image-folder/Vector (1).svg" alt="" /></a>
-            </li>
-            <li>
-              <a href=""><img src="image-folder/Vector.svg" alt="" /></a>
-            </li>
+
+    <div class="card">
+      <div class="margin">
+      <div class="card-img">
+        <img class="card-img1" src="./image-folder/Snapshoot Portfolio (4).png" alt="">
+
+      </div>
+      <div class="project-heading">
+        <h2>Multi-Post <br> Stories </h2>
+        <ul class="canopy">
+          <li class="senior">CANOPY</li>
+          <li><img src="./image-folder/Counter.png" alt=""></li>
+          <li>Back End Dev</li>
+          <li><img src="./image-folder/Counter.png" alt=""></li>
+          <li>2015</li>
+        </ul>
+        <p class="canopy-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="canopy-css">
+          <ul class="canopy-list">
+            <li class="list">HTML</li>
+            <li class="list">CSS</li>
+            <li class="list0">JavaScript</li>
           </ul>
+
         </div>
+        <button class="btn">See project</button>
       </div>
-    </main>
-  </body>
+    </div>
+    </div>
+     </section> 
+
+   <!--ABOUT SECTION-->
+
+
+  <section class="about-section">
+    <div class="main-content1">
+      <div class="author0">
+        <h1>
+          About Myself
+        </h1>
```

Comment:
- [ ] kindly consider having a single h1 on the page cause using only one H1 creates hierarchy and gives your page a more logical structure. It makes it easy for readers (and those using screen readers) to skim and understand what your page is about. Each page should have only one main topic and headline. And that should be your only H1.